package.json

```diff
@@ -11,8 +11,11 @@
     "dojo-release": "dojo-release",
     "intern": "intern",
     "lint": "tslint -p .",
```

Comment:
Most of the other Dojo projects have the lint task set to the following:

```
    "lint:ts": "tslint -p .",
    "lint:prettier": "prettier -l \"{src,tests}/**/*.{ts,tsx}\"",
    "lint": "run-p lint:*",
```

Reply:
Ugh. I did set that but then `npm installed` and overwrote those changes 🤦‍♂️ . Thanks for catching that.